src/java.base/share/classes/java/util/concurrent/CompletableFuture.java

```diff
@@ -2605,8 +2649,8 @@ public int getNumberOfDependents() {
     /**
      * Returns a string identifying this CompletableFuture, as well as
      * its completion state.  The state, in brackets, contains the
-     * String {@code "Completed Normally"} or the String {@code
-     * "Completed Exceptionally"}, or the String {@code "Not
+     * String {@code "Completed normally"} or the String {@code
+     * "Completed exceptionally"}, or the String {@code "Not
```

Comment:
These two looks like a discrepancy between spec and implementation, and it seemed safer to amend the spec rather than the implementation, as code might be relying on *actual behavior* rather than the spec:ed one.

@DougLea You might have some thoughts to offer on this :)